internal/quickstart/show_sdk_instructions.go

```diff
@@ -130,15 +132,15 @@ func (m showSDKInstructionsModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			if err != nil {
 				return m, sendErrMsg(err)
 			}
-			m.viewport.SetContent(md)
+			m.viewport.SetContent(m.headerView() + md)
```

Comment:
This moves the header text into the viewport to simulate scrolling the entire screen.